src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts

```diff
@@ -82,7 +82,17 @@ export class BasicAutocompleteComponent<O, V = O>
     startWith([] as SelectableOption<O, V>[])
   );
   showAddOption = false;
-  private delayedBlur: any;
+
+  get displayText() {
```

Comment:
isn't this going against #1087 and we should rather set a variable instead of a getter?